portal-api/api-tests/v1/users/post-sign-in-link.api-test.js

```diff
@@ -39,8 +39,8 @@ describe('POST /users/me/sign-in-link', () => {
 
   const saltBytes = Buffer.from(saltHexOne, 'hex');
   const signInToken = '0a1b2c3d4e5f67890a1b2c3d4e5f6789';
-  const username = 'TEMPORARY_USER';
-  const userToCreateAsPartiallyLoggedIn = { ...aMaker, username };
+  const temporaryUsernameAndEmail = 'temporary_user@ukexportfinance.gov.uk';
```

Comment:
Could the value be a constant?

Reply:
Noticed some emails are now using constants, but others are hard coded. Should we update all emails to use constants? Maybe another ticket?